pytorch_lightning/loggers/base.py

```diff
@@ -272,10 +272,20 @@ def __init__(self, logger_iterable: Iterable[LightningLoggerBase]):
     def __getitem__(self, index: int) -> LightningLoggerBase:
         return [logger for logger in self._logger_iterable][index]
 
+    def update_agg_funcs(
+            self,
+            agg_key_funcs: Optional[Mapping[str, Callable[[Sequence[float]], float]]] = None,
+            agg_default_func: Callable[[Sequence[float]], float] = np.mean
+    ):
+        [logger.update_agg_funcs(agg_key_funcs, agg_default_func) for logger in self._logger_iterable]
```

Comment:
codefactor marks this as an issue because the list is not use anywhere. perhaps write it without list comprehension.

Reply:
or just `_ = [...]`